index.html

```diff
@@ -50,18 +50,25 @@ <h1 class="text-center font-bold text-blue-800 text-lg">Salón Público de Contr
                         <td class="border px-4 py-2">Jhon Wilder Pariona Vilca</td>
                         <td class="border px-4 py-2">Perú</td>
                         <td class="border px-4 py-2">
-                          <a href="http://github.com/wilderPariona" target="_blank" rel="noopener noreferrer">🐙 wilderPariona</a>
+                            <a href="http://github.com/wilderPariona" target="_blank" rel="noopener noreferrer">🐙
+                                wilderPariona</a>
                         </td>
                     </tr>
                     <tr>
                         <td class="border px-4 py-2">Nathalia</td>
                         <td class="border px-4 py-2">Brazil</td>
                         <td class="border px-4 py-2">@Nthliacc</td>
                     </tr>
+                    <tr class="bg-blue-400">
+                        <td class="border px-4 py-2">Sahib singh</td>
+                        <td class="border px-4 py-2">India</td>
+                        <td class="border px-4 py-2">Github_link</td>
```

Comment:
Hi 👋  a link is necessary

Reply:
please review it again, i have added the link